src/components/formik-santiment-ui/FormikSelect.js

```diff
@@ -14,20 +14,41 @@ const FormikSelect = ({
   return (
     <Field
       name={name}
-      render={({ field, form }) => (
-        <Select
-          placeholder={placeholder}
-          classNamePrefix='react-select'
-          options={options}
-          onChange={value => {
-            form.setFieldValue(name, value)
-            form.setFieldTouched(name, true)
-            onChange && onChange(value)
-          }}
-          value={field.value}
-          {...rest}
-        />
-      )}
+      render={({ field, form }) => {
+        const isValid = !form.errors[name]
+        const customStyles = {
+          control: (base, state) => {
+            return {
+              ...base,
+              borderColor: isValid ? '#ddd' : 'var(--persimmon) !important',
+              '&:hover': {
+                borderColor: isValid ? '#ddd' : 'var(--persimmon) !important'
+              }
```

Comment:
create variable for repeating code, you are constant person

Reply:
Fixed